### PR TITLE
Add more Logs

### LINKED ISSUE #760: Add more logs for transaction at Node Account in the tx services
We need to cover the https://github.com/LimeChain/hashport-validator/blob/main/app/services/scheduled/service.go with more logs related to the printing of Hedera nodes that are hit, not only on the failer. For example https://github.com/LimeChain/hashport-validator/blob/main/app/services/scheduled/service.go#L219
Please check also the app/clients/hedera/client.go

### PR DESCRIPTION
**Pull request by Coiling-Dragon**
Signed-off-by: Stanislav Viyachev <stanislav.viyachev@limechain.tech>

**Detailed description**:

**Which issue(s) this PR fixes**:
Fixes #760

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [ ] Tests updated

### CODE REVIEW

**Review by rokn — COMMENTED** [review]
I don't think we should overburden the schedule service to log info about the transactions. IMO this should be done just in the Hedera client.
Last time we missed those two from the logs, which I thought was the reason for this task:
`func (hc Node) SubmitTopicConsensusMessage(topicId hedera.TopicID, message []byte) (*hedera.TransactionID, error)`
`func (hc Node) SubmitScheduleSign(scheduleID hedera.ScheduleID) (*hedera.TransactionResponse, error)`

I checked them and currently it would require a small logic change to be able to log the node account before executing the transaction, as it's currently directly executing them without freezing. I'm not sure about changing the logic just to be able to log those things. @svetlio8 @trayanm WDYT?

**Review by svetlio8 — APPROVED** [review]

File: app/clients/hedera/client.go [hunk 1]
```diff
@@ -125,28 +125,49 @@ func (hc Node) SubmitScheduledTokenBurnTransaction(tokenID hedera.TokenID, amoun
...
 		SetMaxRetry(hc.maxRetry).
-		Execute(hc.client)
+		FreezeWith(hc.client)
 
 	if err != nil {
 		return nil, err
 	}
 
-	_, err = hc.checkTransactionReceipt(txResponse)
+	hc.logger.Debugf("Submit Schedule Sign, with transaction ID [%s] and Node Account IDs: %v",
```
Comment (rokn) [inline app/clients/hedera/client.go:138]:
```suggestion
	hc.logger.Debugf("Submit Topic Consensus Message, with transaction ID [%s] and Node Account IDs: %v",
```

File: app/clients/hedera/client.go [hunk 2]
```diff
@@ -228,7 +249,7 @@ func (hc Node) submitScheduledTransaction(signedTransaction hedera.ITransaction,
 	scheduledTx = scheduledTx.
 		SetPayerAccountID(payerAccountID).
 		SetScheduleMemo(memo)
-
+
```
Comment (rokn) [inline app/clients/hedera/client.go:252]: nit: whitespace

File: app/services/scheduled/service.go [hunk 3]
```diff
@@ -39,9 +39,14 @@ func New(
...
-	transfers []transfer.Hedera,
-	onExecutionSuccess func(transactionID, scheduleID string), onExecutionFail, onSuccess, onFail func(transactionID string)) {
+		id,
+		nativeAsset string,
+		transfers []transfer.Hedera,
+		onExecutionSuccess func(transactionID, scheduleID string),
+		onExecutionFail,
+		onSuccess, 
+		onFail func(transactionID string),
+	) {
```
Comment (rokn) [inline app/services/scheduled/service.go:49]: nit: This is a formatting only can you revert it?

**Review by trayanm — DISMISSED** [review]

File: app/clients/hedera/client.go (hunk 1 above)
Comment (Coiling-Dragon) [reply app/clients/hedera/client.go]: done

**Review by rokn — APPROVED** [review]

**Pull request merged**